sd-decrypt/decrypt-sd-submission

```diff
@@ -51,6 +52,17 @@ for z in zips:
         zf.extractall(tmpdir + "/extracted/")
     os.unlink(z)
 
+gpgs = glob.glob(tmpdir + "/*gpg")
```

Comment:
we should use `os.path.join` function call here. I can see other parts also where we have used string concatenations, we should use the join call everywhere.